lib/led/rgb.js

```diff
@@ -573,7 +621,7 @@ RGB.prototype[Animation.normalize] = function(keyFrames) {
  */
 
 RGB.prototype[Animation.render] = function(frames) {
-  return this.color(frames[0]);
+  return this.update(frames[0]);
```

Comment:
The animation render method here was calling `color()`, which does a lot of normalization and validation before itself calling `update`.  I've changed this to behave more like the led class, which trusts its animations and calls update directly.